lib/examples/npdsch_enodeb.c

```diff
@@ -218,13 +224,19 @@ void base_init()
     }
   } else {
 #ifndef DISABLE_RF
-    printf("Opening RF device...\n");
-    if (srsran_rf_open(&radio, rf_args)) {
-      fprintf(stderr, "Error opening rf\n");
-      exit(-1);
+    if (strlen(rf_dev) > 0) {
+      if (srsran_rf_open_devname(&radio, rf_dev, rf_args, NOF_TX_ANT)) {
+        fprintf(stderr, "Error opening RF device %s\n", rf_dev);
+        exit(-1);
+      }
     }
+    else
```

Comment:
formatting seems wrong here. Can you use clang-format with the provide file (in the repository root), please?